pkg/services/authn/authnimpl/service.go

```diff
@@ -188,12 +193,14 @@ func (s *Service) Authenticate(ctx context.Context, r *authn.Request) (*authn.Id
 			}
 
 			if identity != nil {
+				s.metrics.successfulAuth.WithLabelValues(item.v.Name()).Inc()
 				return identity, nil
 			}
 		}
 	}
 
 	if authErr != nil {
+		s.metrics.failedAuth.Inc()
```

Comment:
would this give us which provider that failed? Just confused because above we have WithLabelValues

Reply:
i see. 
just a thought i have: in my mind maybe we want failed once to be detailed rather than the successful authentication. this is maybe what we will have soon and this is the stepping stone to have some metrics. wdyt?